sdk/kms/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@mysten/kms",
+	"version": "0.0.1",
+	"description": "A collection of KMS signers for various cloud providers",
```

Comment:
as an aside, I think it'll be a little interesting to see how annoying a unified cloud signer package is given dependencies, but probably fine for now